Upstream: submitted
Source: http://lists.jedsoft.org/lists/slang-users/2016/0000011.html

From 9d876b081a92dda8cfd7921b7842f8089860b562 Mon Sep 17 00:00:00 2001
From: Kylie McClain <somasis@exherbo.org>
Date: Mon, 27 Jun 2016 02:25:37 -0400
Subject: [PATCH] Fix building with many jobs

Without depending on ELFDIR, objects can fail to build if the directory
has not been created yet.
---
 autoconf/aclocal.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autoconf/aclocal.m4 b/autoconf/aclocal.m4
index 383256d..720f460 100644
--- a/autoconf/aclocal.m4
+++ b/autoconf/aclocal.m4
@@ -400,7 +400,7 @@ dnl#}}}
 AC_DEFUN([JD_CREATE_ELFORULE], dnl#{{{
 [
 PROGRAM_ELF_ORULES="$PROGRAM_ELF_ORULES
-\$(ELFDIR)/$1.o : \$(SRCDIR)/$1.c \$(DOT_O_DEPS) \$("$1"_O_DEP)
+\$(ELFDIR)/$1.o : \$(ELFDIR) \$(SRCDIR)/$1.c \$(DOT_O_DEPS) \$("$1"_O_DEP)
 	cd \$(ELFDIR) && \$(ELFCOMPILE_CMD) \$("$1"_C_FLAGS) \$(SRCDIR)/$1.c
 "
 ])
-- 
2.9.0

